workflow/rules/retrieve.smk

```diff
@@ -20,7 +22,13 @@ def get_external_links() -> dict[str,str]:
         "https://dataverse.harvard.edu/api/access/datafile/6040815",
 
         "ember_yearly_electricity_data.csv" :
-        "https://ember-climate.org/app/uploads/2022/07/yearly_full_release_long_format.csv"
+        "https://ember-climate.org/app/uploads/2022/07/yearly_full_release_long_format.csv",
```

Comment:
This has changed (must have been within the last few days)

https://storage.googleapis.com/emb-prod-bkt-publicdata/public-downloads/yearly_full_release_long_format.csv

Reply:
Corrected with [this commit ](https://github.com/OSeMOSYS/osemosys_global/pull/211/commits/922439c14506ade71cb2a9248f7e7fad9d86634a)